prod/docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
       POSTGRES_DB: django
       POSTGRES_PASSWORD: ${DOCKER_POSTGRES_PASSWORD}
     ports:
-      - ${DOCKER_POSTGRES_PORT-5432}:5432
+      - ${DOCKER_POSTGRES_PORT-5432}
```

Comment:
Is this change related to the PR topic? If not, can we isolate it to a separate PR to capture its intent?

Reply:
This was actually done when I was setting up miqademo, and it's purely related to prod setup. We can separate it into another PR, good catch.

Reply:
I put the `:5432` back in, and we can look at this change later on another PR.